packages/icons/src/cdk.svg

```diff
@@ -0,0 +1 @@
.374Zm8.32-.09c.424.18.835.218 1.29.046.638-.243 1.303-.418 1.956-.623.153-.048.303-.103.516-.175-.107-.047-.149-.07-.194-.084-.9-.283-1.8-.57-2.704-.84-.127-.038-.288-.013-.42.027-.835.25-1.666.512-2.498.77-.09.029-.176.065-.346.129l2.4.75Zm-4.37-6.723c.366.116.728.248 1.1.339.18.044.391.052.567 0 .845-.246 1.684-.519 2.525-.784.07-.022.138-.055.27-.11-.8-.25-1.543-.447-2.258-.718a2.202 2.202 0 0 0-1.646.002c-.721.27-1.468.471-2.294.73l1.736.541Zm-1.608 6.705 2.26-.708a42.283 42.283 0 0 0-2.911-.967c-.099-.03-.226.002-.331.034-.844.257-1.685.52-2.527.783-.083.026-.162.065-.303.122.154.055.246.092.34.121.673.212 1.356.396 2.017.64.484.18.94.21 1.455-.025Z"/><path fill="#FFC433" d="M19.246 13.408c0 1.42-.005 2.81.005 4.2.002.212-.059.308-.268.37-.904.27-1.802.558-2.702.838-.106.033-.215.056-.354.091-.007-.135-.017-.234-.017-.334-.001-1.421.004-2.842-.006-4.263-.002-.235.062-.347.295-.417 1-.298 1.995-.616 3.047-.944v.46ZM13.415 7.124l2.13-.66c.008.143.02.242.02.341 0 1.423-.007 2.845.007 4.267.002.259-.086.359-.323.429-.905.269-1.803.56-2.705.84-.087.028-.177.046-.297.076-.007-.121-.017-.22-.017-.318-.001-1.433.005-2.866-.007-4.298-.002-.243.07-.35.307-.403.292-.066.574-.176.885-.274Z"/><path fill="#FFC331" d="M8.486 17.322c0-1.038-.004-2.045.007-3.051.001-.11.07-.29.147-.316 1.032-.341 2.071-.66 3.133-.992.01.11.023.188.023.267.002 1.463 0 2.926.003 4.389 0 .167-.01.287-.213.348-.963.289-1.92.596-2.881.894-.056.018-.119.015-.219.026v-1.566Z"/><path fill="#FA9B35" d="M15.471 14.768v4.14c-.134-.033-.233-.053-.328-.083-.91-.282-1.819-.569-2.73-.848-.164-.05-.263-.106-.261-.312.009-1.513.005-3.027.006-4.54 0-.04.01-.081.02-.158.235.072.46.138.682.208.752.234 1.499.485 2.256.697.282.08.405.2.358.493-.019.121-.003.248-.003.403Z"/><path fill="#FA9A35" d="M11.791 8.177v4.24c-.126-.03-.222-.049-.315-.078l-2.76-.864c-.13-.04-.244-.068-.243-.257.009-1.545.007-3.09.01-4.636 0-.02.01-.039.027-.093.103.026.21.047.312.08.87.27 1.737.556 2.613.81.276.079.413.191.36.487-.017.09-.004.186-.004.31ZM6.687 13.568c.4.124.78.231 1.148.367.089.032.202.159.203.242.01 1.546.005 3.092.003 4.638 0 .02-.014.038-.027.072-.066-.01-.14-.011-.206-.032-.95-.295-1.901-.589-2.848-.897-.091-.03-.217-.152-.218-.232-.01-1.556-.006-3.113-.003-4.67 0-.018.014-.037.033-.085l1.915.597Z"/><path fill="#FFD323" d="m14.959 13.463-2.376-.742c.17-.064.256-.1.346-.128.832-.26 1.663-.521 2.498-.771.132-.04.293-.065.42-.027.905.27 1.804.557 2.704.84.045.014.087.037.194.084-.213.072-.363.127-.516.175-.653.205-1.318.38-1.956.623-.455.172-.866.133-1.314-.054ZM10.589 6.753 8.877 6.22c.826-.26 1.573-.461 2.294-.731a2.202 2.202 0 0 1 1.646-.002c.715.27 1.458.468 2.259.718-.133.055-.2.088-.271.11-.841.265-1.68.538-2.525.785a1.146 1.146 0 0 1-.567 0c-.372-.091-.734-.224-1.124-.347ZM8.98 13.472c-.49.228-.946.197-1.43.018-.66-.244-1.344-.428-2.017-.64-.094-.029-.186-.066-.34-.121.14-.057.22-.096.303-.122.842-.263 1.683-.526 2.527-.783.105-.032.232-.064.33-.034.971.292 1.938.596 2.912.967l-2.284.715Z"/></svg>
```

Comment:
Please run all the svgs through an svg optimizer tool

Reply:
all files already passed through [SVGO](https://jakearchibald.github.io/svgomg/)

Reply:
passed through [SVGO](https://jakearchibald.github.io/svgomg/)
and compressed with [ImageOptim](https://imageoptim.com/mac)